src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Extensions/JsonExtensions.cs

```diff
@@ -212,5 +213,31 @@ public static bool TryConvertTo<TType>(this JToken jobject, out TType result)
             result = default(TType);
             return false;
         }
+
+        /// <summary>
+        /// Converts an <see cref="IDictionary"/> object to <see cref="JObject"/> with an extra level labeled as "value".
+        /// </summary>
+        /// <param name="dict">The <see cref="IDictionary"/> object.</param>
+        /// <param name="keys">The key set to extract from the <see cref="IDictionary"/> object. Default is all keys.</param>
+        /// <returns>The conversion result.</returns>
+        public static JObject ToParameterObject(this IDictionary dict, IEnumerable keys = null)
```

Comment:
The method name ToParameterObject is specific to its use in the policy commandlet.
Since this is a utility method it can be used from elsewhere. Can we use a more generic name like ToJsonObject or ToJObject?